f/connectors/kobotoolbox/kobotoolbox_responses.py

```diff
@@ -145,6 +147,30 @@ def download_form_responses_and_attachments(
     return form_submissions
 
 
+def transform_form_data(form_data):
```

Comment:
```suggestion
def format_geometry_fields(form_data):
```

or some other name that gives a clue that this is about geometry (as opposed to anything else)